readme.md

```diff
@@ -77,7 +77,7 @@ Here I tried to reference the most recent article found on specific software sin
 - [swanky](https://github.com/GaloisInc/swanky) - A suite of rust libraries for secure multi-party computation (currently includes oblivious transfer, garbled circuits, and private set intersection).
 - [TF Encrypted](https://github.com/mortendahl/tf-encrypted/) - 3PC with secret sharing; secure against semi-honest adversaries; focused on TensorFlow-based applications.
 - [TNO-MPC](https://github.com/TNO-MPC/) - Collection of MPC generic software components, procedures, functionalities, and solutions based on secret sharing and (partially) homomorphic encryption developed in Python and Go. | Blogs and principles: [TNO MPC Lab](https://mpc.tno.nl/).
-
+- [JAVASCRIPT-MPC](https://github.com/sjehan/JavascriptMPC) - A golang MPC framework that can compile Javascript files into garbled circuits
```

Comment:
Can you add the framework in alphabetical order? I'd also rename it to `Javascript-MPC` instead of `JAVASCRIPT` written with caps.